<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1477</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="pSlider">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>620</y>
      <width>741</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:3px solid red;
	border-radius:25px;
	background:url(:/leftbutton.png);
	background-position: center;
}

QPushButton:hover{
	border:3px solid black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="play_pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:3px solid red;
	border-radius:35px;
	background:url(:/playbutton.png) no-repeat;
	background-position: center;

}

QPushButton:hover{
	border:3px solid black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next_pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>600</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:3px solid red;
	border-radius:25px;
	background:url(:/rightbutton.png);
	background-position: center;
}

QPushButton:hover{
	border:3px solid black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="curtime_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>620</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>--:--</string>
    </property>
   </widget>
   <widget class="QLabel" name="totoltime_label">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>620</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>--:--</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumSlider">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>620</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playbackmodepushButton">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>613</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid white;
	background-image:url(&quot;:/random.png&quot;) no-repeat;
	background-position:center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="volsw_pushButton">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>615</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid white;
	background-image:url(:/VolOn.png) no-repeat;
	background-position:center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1477</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
